packager/mpd/base/mpd_utils.cc

```diff
@@ -354,6 +359,23 @@ Element GenerateCencPsshElement(const std::string& pssh) {
   return cenc_pssh;
 }
 
+// Extract MS PlayReady Object from given PSSH
+// and encode it in base64
+Element GenerateMsprProElement(const std::string& pssh)
+{
+  std::unique_ptr<media::PsshBoxBuilder> b =
+    media::PsshBoxBuilder::ParseFromBox((const uint8_t*)pssh.data(), pssh.size());
```

Comment:
Can you replace "(const uint8_t*)" with "reinterpret_cast<const uint8_t*>"? We prefer C++ style cast. Same below.

Reply:
Not addressed yet.